.storybook/Container.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Container, Box, theme } from '../src'
+
+storiesOf('Container', module)
+  .add('Default align with theme max width', () => (
+    <Container theme={theme}>
+      <Box p={4} bg='gray0' style={{height: `100vh`}}>
```

Comment:
Just a side note: we've discussed potentially deprecating the numbered color names in favor of `lightGray`, `gray`, and `darkGray` so probably best to use those instead